components/KpiSection.tsx

```
@@ -0,0 +1,49 @@
+    <section className="mb-6 bg-white rounded shadow-sm">
+      <header className="p-4 text-lg font-semibold text-gray-800 border-b border-solid border-b-black border-b-opacity-10">
+        Key Performance Indicators
+      </header>
+      <div className="flex gap-6 px-4 py-6 max-md:flex-col">
+        {kpiItems.map((item, index) => (
+          <KpiItem key={index} title={item.title} value={item.value} />
+        ))}
+      </div>
+      <div className="px-4 pt-0 pb-6">
+        <div
+          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)